packages/vc-handler-ld-local/src/suites/impl/JsonWebSignatureWithRSASupport.ts

```diff
@@ -11,6 +11,10 @@ import * as u8a from 'uint8arrays'
 import { RequiredAgentMethods } from '../../ld-suites'
 
 import { JsonWebKey } from './JsonWebKeyWithRSASupport'
+import { VerificationMethod } from '@sphereon/ssi-types'
+import { JsonWebKey2020, P256Key2021, P384Key2021, P521Key2021 } from '@transmute/web-crypto-key-pair'
+import { Ed25519VerificationKey2018 } from '@transmute/ed25519-key-pair'
+import { EcdsaSecp256k1VerificationKey2019 } from '@transmute/secp256k1-key-pair'
 
 // import { getJwaAlgFromJwk } from '@transmute/web-crypto-key-pair/dist/signatures/jws'
```

Comment:
can it be removed?

Reply:
It's not my line / not commented out by me.
![image](https://github.com/user-attachments/assets/00995a40-01fa-499f-8f07-17af52c1c1b4)

I typically do not remove other people's comments. Perhaps this one is very trivial, maybe it's something that still has to be hooked up in the future, I wouldn't know..
We can create a rule to clean up commented out imports, but until then I do not touch them.